distribution/tools/plugin-cli/build.gradle

```diff
@@ -44,4 +44,18 @@ thirdPartyAudit.onlyIf {
   // FIPS JVM includes manny classes from bouncycastle which count as jar hell for the third party audit,
   // rather than provide a long list of exclusions, disable the check on FIPS.
   project.inFipsJvm == false
-}
\ No newline at end of file
+}
+
+// these two classes intentionally use JDK internal API
```

Comment:
What apis are they using and why? I think that info is important to be in this comment as a future dev evaluating whether this can be removed would need to know it.

Reply:
I added the list of the list of internal classes used. The "why" i think is self-explanatory, unless you mean the technical reasons behind using these internal APIs in the BC code ( assuming there were other alternatives that didn't involve doing so ) but I wouldn't be in the position to answer that.